src/dao/sqlUtils.ts

```diff
@@ -0,0 +1,121 @@
+import { isObject } from '../object'
+import { Operators, SqlErr, SqlError, Where } from './interfaces'
+
+const sqlError = (err: SqlErr): SqlError => {
+  const obj = {
+    code: err.code,
+    message: err.message,
+    sql: err.sql,
+    stack: err.stack
+  }
+
+  if (process.env.IS_OFFLINE) {
+    obj.sql = err.sql
+    obj.stack = err.stack
+    console.error('SQL ERROR:', obj)
+  }
+
+  return { statusCode: 500, error: obj }
```

Comment:
Use the HttpStatuses instead the 500 number

Reply:
Adjusted